docs/en/upgrade.md

```diff
@@ -51,7 +55,70 @@ You can view the status of the stack in the AWS CloudFormation console in the **
 
 You can view the status of the pipeline in the solution console in the **Status** column. After a few minutes, you can receive an Active status.
 
-[quicksight-assets-export]: https://docs.aws.amazon.com/quicksight/latest/developerguide/assetbundle-export.html
+## Post-Upgrade Actions (only applicable to data modeling enabled)
+
+### Upgrade the Data Schema and Out-of-the-box Dashboards
+
+The solution automatically and asynchronously upgrades the views and materialized views used by the dashboard after upgrading the pipeline of the project. The duration of the update depends on the workload of the Redshift cluster and the existing data volume, and can take minutes to hours. You can track the progress in the **Redshift Schemas** section in the **Processing** tab of the Pipeline Detail page. If the post-configuration job fails, you can access the execution of the workflow through its link and rerun the job via **Actions - Redrive** or **New execution** with the input unchanged.
+
+### Migrate the Existing Data (only applicable when upgrading from a version earlier than v1.1.6)
+
+!!! info "Important"
+
+    The data migration process is CPU-intensive. Before starting the migration, ensure that the load on your Redshift is low. It's also advisable to consider temporarily increasing the RPUs of Redshift Serverless or the cluster size when migrating large volumes of data.
+
+    In our benchmark, we migrated 100 million events in 25 minutes using 32 RPUs of Redshift Serverless.
+
+1. Open [Redshift query editor v2][query-editor]. You can refer to the AWS document [Working with query editor v2][working-with-query-editor] to log in and query data using Redshift query editor v2.
+
+    !!! info "Note"
+        You must use the `admin` user or a user with schema (known as the app ID) ownership permission. See this [FAQ][view-schema-in-redshift] for more details.
+
+2. Select the Serverless workgroup or provisioned cluster, `<project-id>`->`<app-id>`->Tables, and ensure that tables for the appId are listed there.
+
+3. Create a new SQL Editor, select your project's schema.
+
+4. Customize the date range as desired, and execute the following SQL in the editor to migrate events from the past 180 days, or any number of days up to the present, to the new tables.
+
+    ```sql
+    -- please replace `<app-id>` with your actual app id
+    -- update the day range based on your needs
+    CALL "<app-id>".sp_migrate_data_to_v2(180);
+    ```
+
+5. Wait for the SQL to complete. The execution time depends on the volume of data in the `events` table.
```

Comment:
it would be better to add a prompt to prevent the user from closing the current window during execution.